libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -189,7 +203,7 @@ void AP_RPM::update(void)
         }
     }
 
-    if (enabled(0) || enabled(1)) {
+    if (enabled(0) || enabled(1) || enabled(2)) {
```

Comment:
See my comment above about logging per instance in AP_BattMonitor as an example.